cmd/proxy/config/apiConfig/v1_0.toml

```diff
@@ -12,6 +12,7 @@
 [APIPackages.about]
 Routes = [
     { Name = "", Open = true, Secured = false, RateLimit = 0 },
+    { Name = "/nodes-versions", Open = true, Secured = false, RateLimit = 0 }
```

Comment:
for reviewers: to be decided if we'll leave this as secured by default

Reply:
it's not a security concern. Can be left unsecured.